content/docs/user-guide/interactivity/robotics/importing-robot/sdformat-sensors.md

```diff
@@ -0,0 +1,66 @@
+---
+linkTitle: SDFormat sensors
+title: SDFormat sensors
+description: Detailed description of support of SDFormat sensors in Robot Importer.
+weight: 100
+---
+
+## Introduction
+
+Robots described in either [SDFormat](http://sdformat.org/), [URDF](http://wiki.ros.org/urdf), or [XACRO](http://wiki.ros.org/xacro) format, can be imported into your O3DE simulation project using the Robot Importer. The tool converts links and joints into O3DE entities and components alongside the parameters defined in the input robot description file. You can find more details about Robot Importer in the [documentation](/docs/user-guide/interactivity/robotics/importing-robot/). 
```

Comment:
It does not just convert links and joints - perhaps this could be reworded?

This suggests that the Robot Importer is doing less than it actually does (colliders, visuals, adding utility components such as ROS 2 Frames, etc).